modules/get-started/pages/whats-new-233.adoc

```diff
@@ -0,0 +1,26 @@
+= What's New in 23.3 Beta
+:description: Summary of new features and updates in the release.
+
+This topic includes new content added in version 23.3 Beta. 
+
+== OIDC authentication
+
+xref:manage:security/console/generic-oidc.adoc[OIDC authentication] provides single sign-on capabilities in Redpanda and Redpanda Console to use external accounts instead of SASL users. With this, you can manage user and client/application credentials in the identity provider, such as Okta, Azure AD, and on-premise Active Directory Federation Service (AD FS).
```

Comment:
OIDC Auth in CONSOLE is not new. This gives the impression it is.  What's new is OIDC for Kafka API, Schema Registry and Admin API.

cc @deniscoady

Reply:
Also Kerberos is a form of external accounts as well